MainModule/Client/Core/Remote.lua

```diff
@@ -505,6 +509,29 @@ return function(Vargs, GetEnv)
 			local ms = ((math.floor((t2-t)*mult+0.5)/mult)*1000)
 			return ms
 		end;
+				
+		FPS = function()
+			local FPS = Remote.Get("FPS")
+			if not FPS then return false end			
+			local startTime = os.clock() 
+			local X = 1 
+			local FPS_Counter = 0
+			
+			local RenderStepped = service.RunService.RenderStepped
+			local Heartbeat = service.RunService.Heartbeat
+
+			local function GetFPS()
+				FPS_Counter+=1
+				if (os.clock() - startTime) >= X  then 
+					local fps = math.floor(FPS_Counter / (os.clock() - startTime))
+					return fps
+					FPS_Counter = 0
```

Comment:
even the pissing linter knows this codes fucked
![image](https://user-images.githubusercontent.com/31691973/183928789-c6d9da04-cc44-41a6-9d76-ffbc6282900f.png)